cns/service/main.go

```diff
@@ -873,6 +892,18 @@ func main() {
 			if err = httpRemoteRestService.SavePnpIDMacaddressMapping(rootCtx); err != nil {
 				logger.Errorf("Failed to fetch PnpIDMacaddress mapping: %v", err)
```

Comment:
are these errors fatal? Seems like we should return them instead of logging them